initializr-generator/src/test/java/io/spring/initializr/generator/buildsystem/maven/MavenBuildWriterTests.java

```diff
@@ -707,13 +707,13 @@ void pomWithCustomVersion() {
 	}
 
 	@Test
-	void powWithDistributionManagementEmpty() {
+	void pomWithDistributionManagementEmpty() {
```

Comment:
Thanks but those unrelated typos should be applied in a different commit. I just did that on `master` in [a polish commit](https://github.com/spring-io/initializr/commit/07b691253526fcf13d138e01f2f708848c08c987).